opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -62,8 +62,8 @@
 
 .. versionadded:: 0.1.0
 .. versionchanged:: 0.3.0
-    `TracerProvider` was introduced and the global ``tracer`` getter was replaced
-    by `get_tracer_provider`.
+    `TracerProvider` was introduced and the global ``tracer`` getter was
+    replaced by `tracer_provider`.
 """
```

Comment:
Not sure, but it might be useful to add a comment about `tracer_provider` being replaced by `get_tracer_provider` here

Reply:
Added in c2feaf1, and I noticed the old methods need to be double-backtick-quoted since the targets don't exist anymore!